src/app/CASEClient.h

```diff
@@ -40,6 +40,8 @@ struct CASEClientInitParams
     // claiming different MRP parameters for the same node.
     Optional<ReliableMessageProtocolConfig> mrpLocalConfig = NullOptional;
 
+    Optional<uint32_t> minimumLitBackoffInterval = NullOptional;
```

Comment:
What does this thing mean?  Needs documentation.

Also: isn't this just a straight-up spec violation if set?  It needs to be clearly documented as such.